Added space to last row of Sample data

tktackett opened the pull request:
A space or escape character is necessary to format the last row Sample data table properly.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
Waiting for Larry's feedback.

Blackmist commented:
ok to merge

tktackett committed:
Added space to last row of Sample data

A space or escape character is necessary to format the last row Sample data table properly.
